03-first-task.Rmd

```diff
@@ -2,4 +2,66 @@
+
+
+
+
+
+
+| Attribute      | Fred Hutch HPC                                                                               | Local Cromwell                    | Local miniwdl                     | Terra                                                                                       |
+|----------------|----------------------------------------------------------------------------------------------|-----------------------------------|-----------------------------------|---------------------------------------------------------------------------------------------|
+| bootDiskSizeGB |                                                                                              |                                   |                                   |                                                                                             |
+| cpu            | Reserve this many cores (acts as a maximum)                                                  | n/a                               |                                   | Request a minimum of this many cores (scales with memory)                                   |
+| disks          |                                                                                              |                                   |                                   |                                                                                             |
+| docker         | Run the task in this Docker image                                                            | Run the task in this Docker image | Run the task in this Docker image | Run the task in this Docker image                                                           |
+| maxRetries     |                                                                                              |                                   |                                   |                                                                                             |
+| memory         | Maximum amount of memory to use                                                              | n/a                               |                                   | Minimum amount of memory to use (scales with CPU)                                           |
+| preemptible    | n/a                                                                                          | n/a                               | n/a                               | Attempt running on a preemptible instance this many times, then switch to a non-preemptible |
+| walltime       | How much  [walltime](https://en.wikipedia.org/wiki/Elapsed_real_time)  to request for a task | n/a                               | n/a                               | n/a                                                                                         |
+
```

Comment:
code snippets with strategies for determining these runtime attributes (disk size, memory, etc)

Reply:
moved to section 05